### PR TITLE
feat(semantic-release): implement for this formula

### LINKED ISSUE #1: Finalise `semantic-release` process
Hey @n-rodriguez, nice work!  You disappeared off Freenode before I got a chance to respond so thought I'd pay you a visit.

I've only had a cursory look over this, so no major feedback as of yet.  I _did_ notice that `semantic-release` hasn't started churning out versions yet.  What's your plan for that?  A couple of thoughts:

1. In all cases, you may want to start the first release manually, otherwise `v1.0.0` will be created.
1. I noticed that you don't have a `.travis.yml` -- I suppose that's because there aren't any tests just yet.  In the meanwhile, you may want a purely `semantic-release` setup, with little or no Kitchen involvement.  You can see how I'm doing that in the [`ssf-formula`](https://github.com/myii/ssf-formula/blob/ac88c9e5125a9d7f570d061df67a30b0981bc9a0/.travis.yml) at the current time.

### PR DESCRIPTION
**Pull request by myii**
* Automated using https://github.com/myii/ssf-formula/pull/55
* Close #1

---

@n-rodriguez Before merging this, the first release should be created.

---

This is working on all instances except for `amazonlinux-2` and `arch-base-latest`:

* https://travis-ci.org/myii/telegraf-formula/builds/592768977
* https://travis-ci.org/myii/telegraf-formula/builds/592791086

To get this working on Arch, the following references are helpful:

* https://wiki.archlinux.org/index.php/Telegraf
* https://aur.archlinux.org/packages/telegraf/
  - This would require `git.latest`
  - Start from https://github.com/saltstack-formulas/iscsi-formula/blob/aa5a94555a46b81ffa2d4efe579deaf0ce0c965b/iscsi/osfamilymap.yaml#L70
* https://www.archlinux.org/packages/?sort=&q=pytoml&maintainer=&flagged=
  - This is already working

### CODE REVIEW

File: telegraf/map.jinja [hunk 1]
```diff
@@ -47,3 +47,18 @@
 %}
 
 {%- set telegraf = config %}
+
+{#- Post-processing for specific non-YAML customisations #}
```
Comment (n-rodriguez) [inline telegraf/map.jinja]: Awesome :+1:

Maybe it could be extracted in a macro? and later in a `utils-formula` ?

**Comment by n-rodriguez** [comment]
> @n-rodriguez Before merging this, the first release should be created.

Maybe we should fork the repo first?

**Comment by myii** [comment]
> Maybe we should fork the repo first?

@n-rodriguez Sure, go for it.  I believe you have the permissions for that, right?

**Comment by n-rodriguez** [comment]
> @n-rodriguez Sure, go for it. I believe you have the permissions for that, right?

Yeah! :) Done: https://github.com/saltstack-formulas/telegraf-formula

**Comment by myii** [comment]
@n-rodriguez Wonderful, but now I have to create my PR again!  I'll see you on the other side.

**Comment by n-rodriguez** [comment]
> @n-rodriguez Wonderful, but now I have to create my PR again! I'll see you on the other side.

Yeah, sorry :/

**Comment by myii** [comment]
Oh, I have to recreate my fork as well... give me a few minutes.

**Comment by myii** [comment]
Closing, since replaced by https://github.com/saltstack-formulas/telegraf-formula/pull/2.

**Pull request closed without merging**